mdetsims/sim_utils.py

```diff
@@ -431,7 +449,7 @@ def get_mbobs(self, return_truth_cat=False):
             mbobs.append(obslist)
 
         if return_truth_cat:
-            return mbobs, truth_cat
+            return mbobs, truth_cat, shear_red
```

Comment:
Add the `shear_red` flag to the truth catalog. Also, rename this flag to something more readable.